src/wizard/javascript/performance-onboarding/javascript/3.verify.md

```diff
@@ -0,0 +1,20 @@
+---
+name: JavaScript
+doc_link: https://docs.sentry.io/platforms/javascript/performance/
+support_level: production
+type: language
+---
+
+#### Verify
+
+Verify that performance monitoring is working correctly by using our [automatic instrumentation](/javascript/performance/instrumentation/automatic-instrumentation/) by simply using your JavaScript application.
```

Comment:
Removing repetition

```suggestion
Verify that performance monitoring is working correctly with our [automatic instrumentation](/javascript/performance/instrumentation/automatic-instrumentation/) by simply using your JavaScript application.
```

Reply:
Github isn't letting me apply this suggestion. So I added it manually.